src/config.rs

```diff
@@ -0,0 +1,44 @@
+use clap::{Parser, Subcommand};
+
+/// Available DNS providers as subcommands, each with its own configuration options
+#[derive(Debug, Subcommand)]
+pub enum DnsProvider {
+    /// Hetzner DNS provider configuration
+    Hetzner(HetznerConfig),
+}
+
+/// Define a struct to hold all command-line arguments
+#[derive(Debug, Parser)]
+#[command(author, about, version)]
+pub struct Config {
+    /// Specifies the address of the Consul server. Defaults to "http://127.0.0.1:8500".
```

Comment:
The help output already contains the default, so including it in the doc comment duplicates the info.